src/app/modules/convai-checker/perspective-status.component.ts

```diff
@@ -189,6 +189,10 @@ export class PerspectiveStatus implements OnChanges, AfterViewInit, AfterViewChe
   private currentStateChangeAnimationId: number = 0;
   private gradientColorScale: string[];
 
+  public percentScore() {
```

Comment:
Move this somewhere below the constructor/ngOn... methods

Reply:
good point!